### ISSUE TITLE
Query bibliography by menu entry

### ISSUE THREAD

**Issue by dasch124** [acdh-oeaw/vicav-vue3#200]
Currently, we can issue a query to the bibligraphy either explicitly by a input field or by a "magic href", e.g. `javascript:getDBSnippet("zotid:EY56FXZI")` c.f. [here](https://tunocent-5b7ead81-4e64-4879-83de-7120e271b4f4.acdh-ch-dev.oeaw.ac.at/?w=W3sieCI6IjAlIiwieSI6IjAlIiwieiI6MTgsIndpZHRoIjoiNTAlIiwiaGVpZ2h0IjoiOTkuOTQlIiwidGFyZ2V0VHlwZSI6IlByb2ZpbGUiLCJ0aXRsZSI6IkEgcHJvZmlsZSBvZiBTaWxpYW5hIEFyYWJpYyIsInBhcmFtcyI6eyJ0ZXh0SWQiOiJwcm9maWxlX3NpbGlhbmFfMDEifX0seyJ4IjoiNTAlIiwieSI6IjAlIiwieiI6MTcsIndpZHRoIjoiNTAlIiwiaGVpZ2h0IjoiOTkuOTQlIiwidGFyZ2V0VHlwZSI6IkJpYmxpb0VudHJpZXMiLCJ0aXRsZSI6IkFiZGVsZmF0dGFoIGFuZCBSaXR0LUJlbm1pbW91biAoMjAyNCkiLCJwYXJhbXMiOnsicXVlcnlTdHJpbmciOiJ6b3RpZDpFWTU2RlhaSSJ9fV0=&a=smart-tile)

It would be great, that we have the same "magic href" from an menu entry, so users can open a pre-factored bibliography query  by clicking on a menu entry. 

User story: In the TUNOCENT instance of VICAV we potentially would like to display three extracts from one bibligraphy: 1) the research bibligraphy used by the project ("everything which has been written on tunesian arabic so far"); 2) the publications which have been written within the project; 3) presentations held by the project members. (the entries of the three sub-bibliographies have been tagged in Zotero)

**Comment by katharinawuensche** [comment]
If I understood it correctly, the `javascript:getDBSnippet("zotid:EY56FXZI")` part is just there for legacy reasons and the magic links are actually built using data attributes coming from the XSLT. After adding the right targetType and params to the menu items in the config, the links should work as expected. Do I see this correctly, @simar0at?

**Comment by simar0at** [comment]
Yes, they will open a BiblQuery. But we I don't know the status of automated query in that window.

**Comment by dasch124** [comment]
Dear @katharinawuensche, are you able to estimate if/when this could work? (Just to know whether we need a quick workaround or not)

**Comment by katharinawuensche** [comment]
From what I have seen in the frontend code, I think it should already be working once the required data attributes are set (see #208). But to be on the safe side, should we discuss this in person, @dasch124 and @ctot-nondef?

**Comment by dasch124** [comment]
great, thanks! Looking from the VICAV-vue3 staging instance, I find links like this:

```
<a class="aVicText" href="javascript:getDBSnippet(&quot;zotid:96YK24BH&quot;)" data-target-type="BiblioEntries" data-label="Siagh (1976)" data-query-string="zotid:96YK24BH">Siagh (1976)</a>
```

So blindly guessing I imagine that the attributes to be add would be `x-data-target-type="BiblioEntries" x-data-label="TUNOCENT Research Bibliography" x-data-query-string="prj:TUNOCENT"` Does this look allright, @ctot-nondef ?

**Comment by ctot-nondef** [comment]
yes, exactly, once these are set the link should work
